### ISSUE TITLE
FEATURE: Add loading indicator option

### ISSUE THREAD

**Issue by foolprooflabs** [prodz18/TiDynamicGrid#3]
I'm currently loading data from a server where a lot of data is returned and it takes a few seconds before it has finished generating all of the views.  None of the views can be clicked until all of them have been created so it would be nice to have a boolean option to show a loading indicator until everything has been rendered.

**Comment by prodz18** [comment]
Yes. That could be awesome.  I'll work on it and include it in the next update.
